src/libraries/System.Private.CoreLib/src/System/IO/FileSystem.Windows.cs

```diff
@@ -513,7 +513,7 @@ internal static void CreateSymbolicLink(string path, string pathToTarget, bool i
                 Span<char> targetPath = MemoryMarshal.Cast<byte, char>(bufferSpan.Slice(printNameNameOffset, printNameNameLength));
                 Debug.Assert(Path.IsPathFullyQualified(targetPath));
                 Debug.Assert((rdb.ReparseBufferSymbolicLink.Flags & Interop.Kernel32.SYMLINK_FLAG_RELATIVE) == 0 ||
-                    targetPath.Length >= 4 && !targetPath.StartsWith(PathInternal.ExtendedPathPrefix) && !targetPath.StartsWith(@"\??\"));
+                    targetPath.Length >= 4 && !targetPath.StartsWith(PathInternal.ExtendedPathPrefix.AsSpan()) && !targetPath.StartsWith(@"\??\".AsSpan()));
```

Comment:
No need to add AsSpan() there is auto cast.

Reply:
This compiles also for `Microsoft.IO.Redist`, which targets net472, it needs netstandard2.0, and these implicit operators are not available there.